DigitalLearningSolutions.Data/Services/CentresService.cs

```diff
@@ -26,5 +27,15 @@ FROM Centres
                 new { centreId }
             );
         }
+
+        public string? GetCentreName(int centreId)

```

Comment:
Hmm.. I think we'd never expect the centre name to be null.. Maybe we could log it if it's null as it is unexpected?

Reply:
It could only be null if there's no centre with that ID, as the field isn't nullable. I'll add a log for that.